.github/workflows/comment.yml

```diff
@@ -1,28 +1,31 @@
-name: CI
+name: Comment on PRs with plugin metadata issues
```

Comment:
feel free to delete this file, it's no longer needed

Reply:
I am not so sure. @Ahmed also mentioned preview is not able to be run by pr from forked repos but this one can. So we can keep it for the plugins developers for the moment?

Reply:
I still think we should delete it, see my comment here https://github.com/aiidateam/aiida-registry/pull/271#issuecomment-1718984077

Reply:
That's fair! I'll remove it.